src/lettuce/plugins/welcome.py

```diff
@@ -0,0 +1,48 @@
+from machine.plugins.base import MachineBasePlugin
+from machine.plugins.decorators import process
+
+
+class WelcomePlugin(MachineBasePlugin):
+    @process("member_joined_channel")
+    async def welcome(self, event):
+        user_id = event['user']
+        channel_id = event['channel']
+
+        welcome_message = f"""
```

Comment:
It'd be more readable to extract the message text to a separate file.

Reply:
thanks for the suggestion , #done